pkgs/development/python-modules/skrl/default.nix

```diff
@@ -1,23 +1,21 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-, fetchpatch
 , pythonOlder
-, pytestCheckHook
 , gym
 , gymnasium
 , torch
 , tensorboard
 , tqdm
 , wandb
 , packaging
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "skrl";
   version = "1.0.0";
-  format = "setuptools";
-
+  pyproject = true;
```

Comment:
`build-system` is implictly `setuptools`, please add it explicitly to `nativeBuildInputs`.

Reply:
Done.